packages/inspector/README.md

```
@@ -1 +1,500 @@
+    - [Vite](#vite)
+    - [Rspack](#rspack)
+    - [Next.js](#nextjs)
+- [Customization](#customization)
+  - [Custom Hotkeys](#custom-hotkeys)
+  - [Controlled Mode](#controlled-mode)
+- [How It Works](#how-it-works)
+- [Supported IDEs/Editors](#supported-ideseditors)
+- [Troubleshooting](#troubleshooting)
+- [Contributing](#contributing)
+- [License](#license)
+- [Acknowledgements](#acknowledgements)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)